lessons/githubPages/lesson.md

```diff
@@ -0,0 +1,226 @@
+
+- **Open Knowledge Foundation: Open Data Handbook** [opendatahandbook.org/guide](http://opendatahandbook.org/guide/en/)  [`github.com/okfn/opendatahandbook`](https://github.com/okfn/opendatahandbook)
+
+- **Grimoire** [ephe.github.io/grimoire/](https://ephe.github.io/grimoire/)  [`ephe.github.io/grimoire/`](https://ephe.github.io/grimoire/)   
+"Grimoire is a mobile-friendly [D&D] 5e spellbook that organizes spell lists by class and level."
+
+- **2048** [gabrielecirulli.github.io/2048/](https://gabrielecirulli.github.io/2048/) [`github.com/gabrielecirulli/2048/`](https://github.com/gabrielecirulli/2048/)
+
+### Naming Conventions
+
+<table class="table table-striped table-hover ">
+  <thead>
+    <tr>
+      <th align="left">Type of GitHub Pages site</th>
+      <th align="left">Pages default domain &amp; host location on GitHub</th>
+      <th align="left">Publishing branch</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td align="left">User Pages site</td>
+      <td align="left"><code>username.github.io</code></td>
+      <td align="left">master</td>
+    </tr>
+    <tr>
+      <td align="left">Organization Pages site</td>
+      <td align="left"><code>orgname.github.io</code></td>
+      <td align="left">master</td>
+    </tr>
+    <tr>
+      <td align="left">Project Pages site owned by a <strong>user account</strong></td>
+      <td align="left"><code>username.github.io/projectname</code></td>
+      <td align="left">gh-pages</td>
+    </tr>
+    <tr>
+      <td align="left">Project Pages site owned by an <strong>organization</strong></td>
+      <td align="left"><code>orgname.github.io/projectname</code></td>
+      <td align="left">gh-pages</td>
+    </tr>
+  </tbody>
+</table>
+
+## Online Editing: Automatic Page Generator
+
+1. Create a new repository ([github.com/new](https://github.com/new))
+  - There are 4 types of Github Pages site (in the table above and on **[GitHub](https://help.github.com/articles/user-organization-and-project-pages/)**)
+  - **following naming conventions is really important!** `.github.io` for user and organizations
+
+2. Navigate to **Settings** and click **Automatic Page Generator**  
+Or just go to generated_pages url for your repository:  
+`https://github.com/<username>/<repository>/generated_pages/new`
+  - Make some edits to the content
+  - Select a theme/layout
+  - Publish!
+
+3. We have a website!  
+`<username/orgname>.github.io/<projectname>/`
+  - Let's check it out
+
+4. Make an update
+
+5. Check out the commit history
+
+## Working Locally: Git and Jekyll
+
+1. First, we need to make sure we have everything installed
+
+~~~
+$ git --version
+git version X.X.X
+$ jekyll -v
+jekyll X.X.X
+~~~
+
+2. We'll have to create (or choose) a repo for a project
+
+~~~
+$ git init first-website
+Initialized empty Git repository in <path/to/current/directory>/first-website/.git/
+~~~
```

Comment:
Tried that, but it doesn't work on my attempt. I could have done it wrong, but either using `~~~` or three backticks doesn't work.

Only two tabs/8 spaces works.